From c7a7d2970163c29da5445df54b0fabe28021b275 Mon Sep 17 00:00:00 2001
From: Miroslav Rezanina <mrezanin@redhat.com>
Date: Wed, 27 May 2015 14:50:33 +0200
Subject: [PATCH 7/8] Remove redhat extensions from qmp-events.txt

Message-id: <021c5d770a75ff1569b7a43a9a08553c9814214c.1432733950.git.mrezanin@redhat.com>
Patchwork-id: 65136
O-Subject: [RHEL-7.2 qemu-kvm PATCH 1/2] Remove redhat extensions from qmp-events.txt
Bugzilla: 1222833
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>
RH-Acked-by: Markus Armbruster <armbru@redhat.com>

From: Miroslav Rezanina <mrezanin@redhat.com>

We document __com.redhat_reason and __com.redhat_debug_info event members
in qmp-events. We describe them as RHEL 7 extension. This is true for
qemu-kvm pacakge only, qemu-kvm-rhev uses upstream solution.

Remove this fields from documentation so users do not expect them so they
avoid issues on update to different version of qemu-kvm.

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 QMP/qmp-events.txt | 17 +----------------
 1 file changed, 1 insertion(+), 16 deletions(-)

diff --git a/QMP/qmp-events.txt b/QMP/qmp-events.txt
index ec4e4b9..7d16a6d 100644
--- a/QMP/qmp-events.txt
+++ b/QMP/qmp-events.txt
@@ -146,28 +146,13 @@ Data:
     "ignore": error has been ignored, the job may fail later
     "report": error will be reported and the job canceled
     "stop": error caused job to be paused
-- "__com.redhat_reason": error reason, this is a RHEL7 extension, it's one of
-  the following (json-string):
-    "eio": errno EIO
-    "eperm": errno EPERM
-    "enospc": errno ENOSPC
-    "eother": any other errno (other than EIO, EPERM, ENOSPC)
-- "__com.redhat_debug_info": RHEL7 extension containing debug information for
-                             humans, applications should NOT read any
-                             information from this member (json-object):
-    - "errno": errno value (json-int)
-    - "message": error message returned by strerror() (json-string)
 
 Example:
 
 { "event": "BLOCK_JOB_ERROR",
     "data": { "device": "ide0-hd1",
               "operation": "write",
-              "action": "stop",
-              "__com.redhat_reason": "enospc",
-              "__com.redhat_debug_info": {
-                  "message": "No space left on device",
-                  "errno": 28 } }
+              "action": "stop" },
     "timestamp": { "seconds": 1265044230, "microseconds": 450486 } }
 
 BLOCK_JOB_READY
-- 
1.8.3.1

